sdks/python/apache_beam/pvalue.py

```diff
@@ -201,6 +201,43 @@ class PDone(PValue):
   pass
 
 
+class PTuple(object):
+  """An object grouping multiple PCollections.
+
+  This class is useful for returning a named tuple of PCollections from a
+  composite.
+  """
+
+  def __init__(self, pcoll_dict):
+    """Initializes this named tuple with a dictionary of tagged PCollections.
+    """
+    self._pcolls = pcoll_dict
+
+  def __str__(self):
+    return '<%s>' % self._str_internal()
```

Comment:
Can we use `str.format()` instead of `%`? Some [example](https://realpython.com/python-f-strings/#option-1-formatting).
`str.format()` is ported back to Python 2.7, so it should be available. But since this is outside of `interactive` package, we can just leave it as it is.

Reply:
Removed this file and changed output to dict.